packages/controller-utils/src/create-service-policy.ts

```diff
@@ -0,0 +1,181 @@
+import {
+  circuitBreaker,
+  ConsecutiveBreaker,
+  ExponentialBackoff,
+  handleAll,
+  retry,
+  wrap,
+  CircuitState,
+} from 'cockatiel';
+import type { IPolicy } from 'cockatiel';
+
+/**
+ * The maximum number of times that a failing action should be re-run before
+ * giving up.
+ */
+export const DEFAULT_MAX_RETRIES = 3;
+
+/**
+ * The maximum number of times that the action is allowed to fail before pausing
+ * further retries.
+ */
+export const DEFAULT_MAX_CONSECUTIVE_FAILURES = (1 + DEFAULT_MAX_RETRIES) * 3;
```

Comment:
Question: Should this be dynamically calculated based on the `maxRetries` passed? I took this from the existing implementations but I don't know if it was intentional that it was static.